### PR TITLE
Don't use restart for mysql brew service

### PR DESCRIPTION
**Pull request by tarebyte**

### CODE REVIEW

File: cmd/brew-upgrade-mysql [hunk 1]
```diff
@@ -59,7 +59,9 @@ if [[ -d $(brew --prefix mysql@5.6) || -d $(brew --prefix mysql@5.7) ]]; then
 
   mysql_upgrade
```
Comment (ggunson) [inline cmd/brew-upgrade-mysql]: Do we need the $(brew prefix) stuff here to get the path?

Comment (tarebyte) [reply cmd/brew-upgrade-mysql]: @ggunson it's an alias for the function above https://github.com/github/homebrew-bootstrap/pull/49/files#diff-8bc3b644981611d8b5b661ba420e8c64R5

Comment (tarebyte) [reply cmd/brew-upgrade-mysql]: @ggunson I've changed the function name in d794c45 to clear things up for the reader

**Comment by ggunson** [comment]
:+1: from me

File: cmd/brew-upgrade-mysql [hunk 2]
```diff
@@ -43,11 +45,22 @@ EOM
 if [[ -d $(brew --prefix mysql@5.6) || -d $(brew --prefix mysql@5.7) ]]; then
   if [[ -d $(brew --prefix mysql@5.6) ]]; then
     brew services stop mysql@5.6 || true
+
+    while ! [[ "$(pidof mysqld)" == "" ]]; do
```
Comment (MikeMcQuaid) [inline cmd/brew-upgrade-mysql]: @tarebyte FYI: `pidof` isn't a command provided by macOS. On my machine I get: `vendor/homebrew-bootstrap/cmd/brew-upgrade-mysql: line 49: pidof: command not found`. Instead, how about `pgrep mysqld`?

**Comment by ggunson** [comment]
I think it's on me to do some fixes to this. On top of what @MikeMcQuaid found re: pidof, this shouldn't restart mysql if the host was properly upgraded before (which is knowable from a file in the datadir).

**Pull request merged**